### PR TITLE
Add wallpapper

### PR DESCRIPTION
**Pull request by vpeschenkov**
## Project URL
https://github.com/mczachurski/wallpapper
## Category
Images
## Description
wallpapper is a console application for creating dynamic wallpapers for Mojave. 
## Checklist

- [x] Only one project/change is in this pull request
- [x] Addition in alphabetical order
- [x] Appropriate language icon(s) added if applicable
- [x] Has a commit from less than 2 years ago
- [x] Has a **clear** README in English

### CODE REVIEW

**Comment by serhii-londar** [comment]
@vpeschenkov Thanks for your contribution! 👍 
I've done a lot of updates last time, that's why your PR have conflicts. Can you resolve them?
I think the 'Terminal' category would be better? What do you think?

**Comment by vpeschenkov** [comment]
@serhii-londar Sure! Just didn't notice about that category...

File: README.md [hunk 1]
```diff
@@ -527,6 +527,7 @@ You can see in which language an app is written. Currently there are following l
 - [iTerm 2](https://github.com/gnachman/iTerm2) - Terminal emulator for macOS that does amazing things. ![objective_c_icon]
 - [Kitty](https://github.com/kovidgoyal/kitty) - Cross-platform, fast, feature full, GPU based terminal emulator. ![c_icon] ![python_icon]
 - [OpenTerminal](https://github.com/es-kumagai/OpenTerminal) - App for macOS that opens a new Finder window and changes the current directory to the folder launched by the app. ![swift_icon]
+- [wallpapper](https://github.com/mczachurski/wallpapper) - wallpapper is a console application for creating dynamic wallpapers for Mojave. ![SwiftIcon]
```
Comment (serhii-londar) [inline README.md]: Please change from ![SwiftIcon] to ![swift_icon]

Comment (vpeschenkov) [reply README.md]: Done

**Comment by serhii-londar** [comment]
@vpeschenkov Looks good! Thank you! 👍

**Pull request merged**